### DISCUSSION TITLE
How to connect to Homassistant VM in a local Network

### DISCUSSION THREAD

**Discussion by appsheld** [wg-easy/wg-easy#325] [Q&A]
Because my Provider dont forward incomming Connections I want connect to the Web-Interface of my Homeassistant server over a wiregard Server. This Server is a Ubuntu 20.04 VM with static IP to the Internet. I try to open port 8123 to server 10.8.0.2 on wg-easy.

docker run -d \
  --name=wg-easy \
  -e WG_HOST=123.44.194.123 \
  -e PASSWORD=geheim \
  **-e WG_PRE_UP='iptables -t nat -A PREROUTING -d 123.44.194.123 -p tcp --dport 8123 -j DNAT --to-destination 10.8.0.2' \**
  -v ~/.wg-easy:/etc/wireguard \
  -p 51820:51820/udp \
  -p 51821:51821/tcp \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_MODULE \
  --sysctl="net.ipv4.conf.all.src_valid_mark=1" \
  --sysctl="net.ipv4.ip_forward=1" \
  --restart unless-stopped \
  weejewel/wg-easy

The connection from 10.8.0.2 to wg-easy is established and traffic to Internet is possible but no inbound connection over port 8123 from the Internet. Whats is wrong?

**Comment by archont94** [comment]
Maybe you need to expose port 8123 as well? Otherwise docker host will not pass it to wg-easy container.
